comms/src/connection_manager/manager.rs

```diff
@@ -124,6 +124,20 @@ impl ConnectionManager {
                         self.initiate_peer_connection(peer)?
                     },
                     // Already have an active connection, just return it
+                    PeerConnectionState::Listening(Some(address)) => {
+                        debug!(
+                            target: LOG_TARGET,
+                            "Listening on {} for NodeId={}...", address, peer.node_id
```

Comment:
Possibly ambiguous log message:
Am I listening for a specific different Node? (In which case mybe "Waiting for [NodeID] to connect to me at {}")
Or is this logging my node ID as part of the listening message? (In which case "[NodeId]: Listening for inbound connections")

Reply:
Thanks - the former :) Fixed